frontend/src/app/login/page.tsx

```
@@ -0,0 +1,203 @@
+      <header className="fixed w-full top-0 z-50 flex items-center justify-between whitespace-nowrap border-b border-white/10 bg-black px-6 py-3 lg:px-20 shadow-md">
+        <NextLink href="/" className="flex items-center gap-3 text-white hover:opacity-80 transition-opacity">
+          <div className="size-7 rounded-lg bg-primary/20 flex items-center justify-center text-primary-light">
+            <span className="material-symbols-outlined !text-xl">bookmarks</span>
+          </div>
+          <h2 className="text-white text-xl font-bold leading-tight tracking-tight">SearchWeb</h2>
+        </NextLink>
+        <div className="flex flex-1 justify-end gap-8">
+          <div className="hidden items-center gap-9 md:flex">
+            <a className="text-white/90 hover:text-white text-sm font-medium transition-colors" href="/#features">기능</a>
+            <a className="text-white/90 hover:text-white text-sm font-medium transition-colors" href="/#pricing">가격</a>
+            <a className="text-white/90 hover:text-white text-sm font-medium transition-colors" href="#">문의하기</a>
```

Biome (2.4.4)

[error] Provide a valid value for the attribute href.

(lint/a11y/useValidAnchor)